docs/setup/native.md

```diff
@@ -6,18 +6,24 @@ Vets API requires:
 - PostgreSQL 15.x (including PostGIS 3)
 - Redis 6.2.x
 
-  The most up-to-date versions of each key dependency will be specified in the `docker-compose.yml` [file](https://github.com/department-of-veterans-affairs/vets-api/blob/master/docker-compose.yml) and the `Dockerfile`.
+The most up-to-date versions of each key dependency will be specified in the `docker-compose.yml` [file](https://github.com/department-of-veterans-affairs/vets-api/blob/master/docker-compose.yml) and the `Dockerfile`.
 
-  We suggest using a Ruby version manager such as [`rbenv`](https://github.com/rbenv/rbenv#installation), `asdf`, `rvm`, or `chruby` to install and maintain your version of Ruby.
+## Installing Ruby Version Manager
```

Comment:
Maybe: Installing a Ruby version manager?

The title sounds like it's still suggesting RVM